hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/Dispatcher.java

```diff
@@ -532,6 +532,25 @@ public long getNumBytes(StorageGroup storage) {
       }
       return block.getNumBytes();
     }
+
+    public void setIndices(byte[] indices) {
+      this.indices = indices;
+    }
+
+    public void adjustIndices(List<Integer> list) {
```

Comment:
Please provide comments on what this method does.

Reply:
add the comments like this.
```java
    /**
     * Adjust EC block indices，it will remove the element of adjustList from indices.
     * @param adjustList the list will be removed from indices
     */
    public void adjustIndices(List<Integer> adjustList) {
```